arcsi/view/archive.py

```diff
@@ -15,3 +16,33 @@ def archive():
     return render_template(
         "archive/list_archive.html"
     )
+
+@router.route("/api_key_admin")
+def api_key_admin():
+    username = "endre"
+    #user = guard.user_class.lookup(username)
+    #guard._check_user(user)
+    password = "YhhU8Z4W2dVbKNf"
+    #hashed_password = guard.hash_password(password)
+    #app.logger.error("password: " + password)
+    #app.logger.error("hashed_password: " + hashed_password)
+    #return jsonify(hashed_password)
+    user = guard.authenticate(username, password)
+    
+    ret = {"access_token": guard.encode_jwt_token(user)}
+    return (jsonify(ret), 200)
+
+@router.route("/api_key_nemadmin")
```

Comment:
I like this way of testing behaviour by creating test endpoint but please push it with a simple standard name. A hunglish out of schema name is harder to use and easy to miss when cleaning up.

Reply:
These two APIs are really just for testing the functionalities, also they are containing hardcoded values.